out/schema.json

```diff
@@ -242,6 +242,56 @@
           "cType": "h"
         }
       ]
+    },
+    "OCT_GPS_2": {
+      "id": 8,
+      "header": [
+        {
+          "name": "valid_bitfield",
+          "cType": "H"
+        },
+        {
+          "name": "epoch",
+          "cType": "d"
+        }
+      ],
+      "body": [
+        {
+          "name": "OCT_GPS_TimeUCT",
+          "cType": "I"
+        },
+        {
+          "name": "OCT_GPS_NumSats",
+          "cType": "B"
+        },
+        {
+          "name": "OCT_GPS_HDOP",
+          "cType": "B"
+        }
+      ]
+    },
+    "OCT_GPS_1": {
```

Comment:
The name of this message in the DBC could be made more descriptive. Something like `OCT_GPS_LATLONG`. Similar for the other message, but less sure on the name there.